src/test/java/SetTest.java

```diff
@@ -0,0 +1,44 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@DisplayName("Set 자료형 관련 테스트")
+public class SetTest {
+    private Set<Integer> numbers = new HashSet<>();
+
+    @BeforeEach
+    void setUp() {
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set의 크기를 확인한다.")
+    @Test
+    void checkSetSize() {
+        int result = numbers.size();
+        Assertions.assertThat(result).isEqualTo(3);
```

Comment:
```suggestion
        Assertions.assertThat(result).hasSize(3);
```
컬렉션을 쉽게 테스트할 수 있게 메서드를 제공하고 있습니다.